src/pyuvdata/telescopes.py

```diff
@@ -709,7 +1021,7 @@ def from_known_telescopes(
         run_check: bool = True,
         check_extra: bool = True,
         run_check_acceptability: bool = True,
-        known_telescope_dict: dict = _KNOWN_TELESCOPES,
+        **kwargs,
```

Comment:
This one is also not documented.

Reply:
Added in 1b8340609b362fc9b9225a3eb7e9535b7793952d.